vue/src/i18n/index.js

```diff
@@ -0,0 +1,19 @@
+import Vuex from 'vuex';
+
+// Import translations
+import english from './translations/english';
```

Comment:
I'd identify the languages by ietf language codes, dialects matter. What do you think @labrys